developer_manual/app/custom-cache-backend.rst

```diff
@@ -0,0 +1,246 @@
+|                                                            | child entries in case of moving a folder.                        |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``moveFromFolder($sourceCache, $sourcePath, $targetPath)`` | Move a file or folder from a different cache instance.           |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``clear()``                                                | Remove all entries from the cache.                               |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``getStatus($file)``                                       | Get the scanned status of a file or folder.                      |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``search($pattern)``                                       | Search the cache for a file or folder where the filename matches |
+|                                                            | ``$pattern``, SQL style wildcards are used in the pattern.       |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``searchByMime($mimetype)``                                | Search a file or folder with a matching mimetype, both full      |
+|                                                            | mimetypes ('text/plain') and mimetype groups ('text') should be  |
+|                                                            | supported as search option.                                      |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``correctFolderSize($path)``                               | Recalculate the size of a folder and all parent folders.         |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``calculateFolderSize($path)``                             | recalculate the size of a single folder.                         |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``getAll()``                                               | Get the file id for all files and folder in the cache            |
++------------------------------------------------------------+------------------------------------------------------------------+
+| ``getIncomplete()``                                        | Get one folder which a status of ``Cache::SHALLOW``.             |
```

Comment:
Not sure what "Get one folder which a status of..." means?
Maybe something to do with "with a status of..." or?